### ISSUE TITLE
Does not work with Unicode strings

### ISSUE THREAD

**Issue by makyen** [psyked/LZW-js#1]
This package does not work with strings containing Unicode characters (i.e. characters which require two bytes to represent them). At a minimum, it would be helpful to explicitly mention this restriction in the README.md file.

Passing it a string containing such characters will result in the decoded string being corrupted.
For example, the string : `'йд'` will be corrupted. when passed through encode->decode.

In other words:

    var test = 'йд';
    console.log(LZW.decode(LZW.encode(test)) === test); //This is false.

**Comment by psyked** [comment]
Thanks for getting in touch @makyen. Unicode characters weren't really considered when this was initially put together so it was overlooked. I'll update the README accordingly.
